tools/applicationinsights-web-snippet/src/type.ts

```diff
@@ -0,0 +1,60 @@
+import { IConfiguration } from "@microsoft/applicationinsights-web";
+export interface ISnippetConfig {
+    src: string;
+    name?: string;
+    ld?: number;
+    useXhr?: boolean;
+    crossOrigin?: string;
+    onInit?: any;
+    cfg: {
+        connectionString: any;
+    };
+}
+
+export interface AIConfig extends IConfiguration {
+    autoExceptionInstrumented?: boolean;
```

Comment:
Oh, interesting....

These are not "really" configuration items, but "hacks" to store some information indirectly into the configuration...

Reply:
autoExceptionInstrumented - Lets define this in the IConfiguration as it is used via the AnalyticsPlugin.
url - Lets remove (I have another comment where it's used)